### DISCUSSION TITLE
Fusion360 files

### DISCUSSION THREAD

**Discussion by nicholasmparker** [WhereslyClock/MyWhereslyClock#11] [General]
Would you be open to including the files?  I'd like to modify your base to use some sail winch servos I have and its difficult from the stl without rebuilding.. At least for me.

Thank you!

**Comment by WhereslyClock** [comment]
Hi Sorry i dont monitor this account,  If you still intersted   email
***@***.*** with your email and remind me why,  and I can invite you
to the fusion project,  itll give you read rights and you can figure out
how to export it,

**Comment by WhereslyClock** [comment]
hit me up on bluesky, binaryjam . tocker .co .uk   if you didnt get a direct mail with email
